modules/core/src/transactionBuilder.ts

```diff
@@ -920,10 +920,10 @@ exports.signTransaction = function(params) {
     }
 
     const chainPath = currentUnspent.chainPath;
-    if (rootExtKeyPath) {
-      const subPath = keychain.walletSubPath || '/0/0';
-      const path = keychain.path + subPath + chainPath;
-      privKey = rootExtKeyPath.deriveKey(path);
+    if (rootExtKey) {
+      let { walletSubPath = '/0/0' } = keychain;
```

Comment:
This can go back to a `const` now I believe.

Reply:
yes